test/autotest.py

```diff
@@ -543,8 +547,9 @@ def testRestart():
 
     #start user programs
     for cmd in cmds:
-      procs.append(launch(BIN+"dmtcp_launch "+cmd))
-      sleep(S*SLOW)
+      procs.append(runCmd(BIN+"dmtcp_launch "+cmd))
+    #wait and see if process will die shortly after checkpoint
+    sleep(S*SLOW)
```

Comment:
Again, as with other `sleep` statements, this should go _after_ the following `WAITFOR` statement.